docs/misc/dvd-remux.md

```diff
@@ -0,0 +1,670 @@
+# Remuxing DVDs
+
+When working on DVDs,
+it helps a lot to have a clean mux
+of each episode ready.
+To do this,
+you will need to remux the DVDISO,
+and somehow split the remux into individual episodes.
+This guide will cover different methods to accomplish this,
+as well as cover some additional information
+that's useful if you want to release a DVD remux.
+
+## Understanding DVD Structure
+
+There are multiple ways to remux a DVDISO.
+However, it's also important
+to understand _what_ you are remuxing.
+DVDISOs are made up of the following:
+
+- **Titles**: The main content units on a DVD. Each title typically contains one episode, movie, or bonus feature. These may also contain multiple episodes in one big title.
+- **Angles**: Alternative video streams within a title, often used for different versions of the same content (such as theatrical and director's cut versions).
+- **Chapters**: Markers within a title that allow skipping to specific points, like scene transitions or commercial break points.
+
+Each of these elements can be important
+depending on what you're trying to preserve.
+For a basic episode remux,
+you'll typically want to focus on the main title and chapters.
+However, for a complete preservation,
+you might want to capture all available angles as well.
+
+??? question "What are PGCs?"
+    PGCs (Program Chains) are part of the DVD video structure hierarchy.
+    At the lowest level, multiple VOBUs (Video Object Units) form a cell.
+    One or more cells make up a program,
+    and one or more programs form a PGC.
+    Programs can be identified as PTTs (Parts of Title),
+    also commonly known as chapters.
+    A title on a DVD consists of one or more PGCs,
+    while a menu is made up of a single PGC[^pgc-references].
+
+[^pgc-references]: For more information,
+    see [this page](https://dvd.sourceforge.net/dvdinfo/pgc.html),
+    [this page](http://www.mpucoder.com/DVD/),
+    and [this page](https://en.wikibooks.org/wiki/Inside_DVD-Video/MPEG_Format#VOBU,_Cell,_Program,_PGC).
```

Comment:
Links like `[here]` or `[this page]` are generally discouraged, I'd rather paste the links in full or have the link text be a short description like `[the DVD-Video Information Page]`.